### DISCUSSION TITLE
Using an exported dataset from piper-recording-studio (ljspeech) with piper TRAINING.md - no config.json?!?!?

### DISCUSSION THREAD

**Discussion by mylegitches** [rhasspy/piper#651] [Q&A]
I have made an issue in [rhasspy/piper-recording-studio](https://github.com/rhasspy/piper-recording-studio/issues/23).

I am posting here since it is dealing with both repos, in a sense.

I want to train using a particular .ckpt file with my own recorded voice, which I have done using piper-recording-studio to record and export the dataset.

I believe all of this is in order.

My issue is that when I go to attempt training using the command:

```
python3.8 -m piper_train \
    --dataset-dir /home/adam/piper/converted_to_ljspeech/ \
    --accelerator 'gpu' \
    --devices 1 \
    --batch-size 32 \
    --validation-split 0.0 \
    --num-test-examples 0 \
    --max_epochs 10000 \
    --resume_from_checkpoint //home/adam/piper/ckpt/adam_base.ckpt \
    --checkpoint-epochs 1 \
    --precision 32
```

It seems as though it is looking for a config.json file in dataset directory.  I don't have this file, nor am I sure where to obtain it.

Is it possible to do what I am attempting?  I would imagine so...

Please help!

**Comment by EvanMason489** [comment]
Have you done the preprocessing step? I think that might be where the config.json comes from?

**Reply by mylegitches** [reply]
Oof!  Sorry for late reply.  At this point the end of November was a long time ago.  I went back with a different approach and got where I needed to be.  Thanks!
